samples/snippets/src/main/java/dialogflow/cx/ListPages.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dialogflow.cx;
+
+// [START dialogflow_cx_list_pages]
+import com.google.cloud.dialogflow.cx.v3.ListPagesRequest;
+import com.google.cloud.dialogflow.cx.v3.ListPagesRequest.Builder;
+import com.google.cloud.dialogflow.cx.v3.Page;
+import com.google.cloud.dialogflow.cx.v3.PagesClient;
+import java.io.IOException;
+
+public class ListPages {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String agentId = "my-agent-id";
+    String flowId = "my-flow-id";
+    String location = "my-location";
+
+    listPages(projectId, agentId, flowId, location);
+  }
+
+  // DialogFlow API List Pages Sample.
+  // Lists all pages from the provided parameters 
+  public static String listPages(String projectId, String agentId, String flowId, String location)
+      throws IOException {
+    PagesClient client = PagesClient.create();
+    Builder listRequestBuilder = ListPagesRequest.newBuilder();
+
+    listRequestBuilder.setParent(
+        "projects/"
+            + projectId
+            + "/locations/"
+            + location
+            + "/agents/"
+            + agentId
+            + "/flows/"
+            + flowId);
+    listRequestBuilder.setLanguageCode("en");
+
+    String result = "";
+
+    // Make API request to list all pages in the project
+    for (Page element : client.listPages(listRequestBuilder.build()).iterateAll()) {
+      result += element.toString() + "\n";
```

Comment:
This doesn't seem realistic but purely for testing purposes? Most samples print out on each iteration like a name

Reply:
yea i did this when i was trying to get around returning the response but i'll change this to the normal print